.github/workflows/ci-testing.yml

```diff
@@ -26,13 +26,20 @@ jobs:
           pip install .[dev]
 
       - name: Run the test suite \w coverage 
-        if: ${{ matrix.python-version }} == "3.11"
+        if: contains(matrix.python-version, '3.12')
         run: pytest --cov=aiidalab_alc --cov-report xml:coverage.xml
       
       - name: Run the test suite without coverage 
-        if: ${{ matrix.python-version }} != "3.11" 
+        if: ${{ matrix.python-version }} != '3.12'
```

Comment:
I think this needs to be a `! contains...` because currently the tests are running twice for Python 3.12.